Code Quality/Tools.md

```diff
@@ -162,15 +160,14 @@ Even though [TSLint](https://palantir.github.io/tslint/) is deprecated, it is st
 
 Infinum created various rulesets for ESLint and TSLint which you can check out [here](https://github.com/infinum/js-linters). We encourage you to use some of these presets, as per your needs:
 
-  - [Base ESLint config](https://github.com/infinum/js-linters/blob/master/eslint-config/README.md)
-  - [React ESLint config](https://github.com/infinum/js-linters/blob/master/eslint-config-react/README.md)
-  - [Angular TSLint config](https://github.com/infinum/js-linters/tree/master/tslint-config-angular)
+-   [Base ESLint config](https://github.com/infinum/js-linters/blob/master/eslint-config/README.md)
```

Comment:
Are you sure this change will not mess up rendering when deployed? I know that our markdown-to-html is a bit fussy about indentations and lists. Can you please merge to staging and check?

Reply:
I'll check!

Reply:
Works. This is how prettier formats it.